sklearn/feature_selection/_univariate_selection.py

```diff
@@ -259,23 +259,18 @@ def f_regression(X, y, *, center=True):
 
     Returns
     -------
-    F : array, shape=(n_features,)
-        F values of features.
-
-    pval : array, shape=(n_features,)
-        p-values of F-scores.
+    corr : array, shape=(n_features,)
```

Comment:
Since the function is simple, I would suggest to change the code to call `corr` -> `corr_coef` (or even `correlation_coefficient`)

```suggestion
    corr_coef : ndarray of shape (n_features,)
```